index.html

```diff
@@ -9,16 +9,24 @@
     <title>Portfolio</title>
   </head>
   <body>
+    <!-- Header section -->
     <header class="header">
       <nav class="nav container-padding-x container">
         <a class="logo" href="#" target="_blank">My Logo</a>
+        <div class="menu">
+          <a class="link" href="#portfolio" target="_self">Portfolio</a>
+          <a class="link" href="#about" target="_self">About</a>
+          <a class="link" href="#contact" target="_self">Contact</a>
+        </div>
         <i class="fa-solid fa-bars"></i>
       </nav>
```

Comment:
- [ ] Make sure the space between the `logo` and the `nav-list` should stick to the Figma design, It's an important part of the requirement. Kindly fix that. :+1: 

Your design

![Screenshot from 2022-12-08 23-14-30](https://user-images.githubusercontent.com/80458199/206572310-6ad6d292-8908-48b1-baa9-2817ad8cd508.png)

Figma Design

![Screenshot from 2022-12-08 23-16-10](https://user-images.githubusercontent.com/80458199/206572370-c49e3810-b756-4ac0-95c1-246651020e76.png)